grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/ArmeriaMessageFramer.java

```diff
@@ -104,19 +110,23 @@ public ArmeriaMessageFramer(ByteBufAllocator alloc, int maxOutboundMessageSize)
      * Writes out a payload message.
      *
      * @param message the message to be written out. Ownership is taken by {@link ArmeriaMessageFramer}.
+     * @param webTrailers tells whether the payload is web trailers
      *
      * @return an {@link HttpData} with the framed payload. Ownership is passed to caller.
      */
-    public HttpData writePayload(ByteBuf message) {
+    public HttpData writePayload(ByteBuf message, boolean webTrailers) {
```

Comment:
nit: How about introducing a new method such as `writeWebTrailers(ByteBuf message)` so that you don't need to put `false` every where?

Reply:
That's a good idea. Added!